tests/aws/services/ssm/test_ssm.py

```diff
@@ -159,6 +159,20 @@ def test_get_parameter_by_arn(self, create_parameter, aws_client, snapshot, clea
         snapshot.match("Parameter", parameter_by_arn)
         snapshot.add_transformer(snapshot.transform.key_value("Name"))
 
+    @markers.aws.validated
+    def test_get_parameter_history(self, create_parameter, aws_client):
+        param_name = f"param-{short_uid()}"
+        create_parameter(Name=param_name, Value="test", Type="String")
+        create_parameter(Name=param_name, Value="test2", Type="String", Overwrite=True)
+        create_parameter(Name=param_name, Value="test3", Type="String", Overwrite=True)
+
+        response = aws_client.ssm.get_parameter_history(Name=param_name)
+        assert len(response["Parameters"]) == 3
+        assert response["Parameters"][0]["Value"] == "test"
+        assert response["Parameters"][0]["Version"] == 1
+        assert response["Parameters"][2]["Value"] == "test3"
+        assert response["Parameters"][2]["Version"] == 3
```

Comment:
Typically we'd use our snapshot library in these cases, which will  also catch changes to the API if we re-validate the case in the future. You can find more information on this in our [contribution guidelines](https://github.com/localstack/localstack/tree/master/docs/testing/parity-testing).

Please let me know if you'd like to tackle this yourself, otherwise I can also generate this for you! :+1:

Reply:
Thanks! I will add snapshot test soon :)